packages/loaders/src/Loader.ts

```diff
@@ -155,6 +155,15 @@ export class Loader extends ResourceLoader
 
         return Loader;
     }
+
+    /**
+     * All the resources for this loader keyed by name.
+     *
+     * @member {object<string, ILoaderResource>}
+     */
+    resources: {
+        [key: string]: import('@pixi/loaders').ILoaderResource;
```

Comment:
Referencing ILoaderResource from module itself, are we sure just `ILoaderResource` wont work?

Reply:
Please don't use this import syntax here. Import type at the start of the file to import the interface. 

I thought that resources was a property of resource-loader. Why is it not inherited?